scaling_performance/optimizing_storage.adoc

```diff
@@ -106,9 +106,30 @@ The following table summarizes the recommended and configurable storage technolo
 A scaled registry is an {product-title} registry where three or more pod replicas are running.
 ====
 
+[IMPORTANT]
+====
+For metrics you can use RWO or RWX as a single shared volume. However, using RWX
+is an anti-pattern.
+
+For logging, you are required to have one volume per `logging-es` DC. However,
+using RWX is an anti-pattern. 
+====
```

Comment:
Thank you, @sferich888 and @sjwalter2! I added a warning following the table. It needs a bit more word-smithing, but is this along the lines of what you're looking for?

For the  "However, using RWX is an anti-pattern." clause, would it be better written, "However, using RWX can cause an anti-pattern."?

Reply:
@sferich888 @sjwalter2 - *bump*. We want to close this as soon as possible.